Documentation/contributing/development/dev_setup.rst

```diff
@@ -579,12 +579,34 @@ Debugging
 Datapath code
 ^^^^^^^^^^^^^
 
-The tool ``cilium monitor`` can also be used to retrieve debugging information
-from the eBPF based datapath. Debugging messages are sent if either the
-``cilium-agent`` itself or the respective endpoint is in debug mode. The debug
-mode of the agent can be enabled by starting ``cilium-agent`` with the option
-``--debug`` enabled or by running ``cilium config debug=true`` for an already
-running agent. Debugging of an individual endpoint can be enabled by running
+The image below shows the options that could be used as startup options by
+``cilium-agent`` (see upper blue box) or could be changed at runtime by running
+``cilium config <option(s)>`` for an already running agent (see lower blue box).
+For instance, if you want to enable the debug mode (including the load
+balancer), you may start ``cilium-agent`` with ``--debug-verbose=datapath`` or
+run ``cilium config debug=true debugLB=true`` for an already running agent. For
+each option, there is a helper function which can be used by the developer to
+print debugging information in the tool ``cilium monitor``: ``cilium_dbg()`` for
+``DEBUG`` and ``cilium_dbg_lb()`` for ``DEBUG_LB``. 
+
+.. image:: _static/cilium-debug-datapath-options.svg 
```

Comment:
Nit: Any chance we could have lighter backgrounds (or white foreground) to improve readability for the dark-grey boxes?

Reply:
Let me know if it is as you expected.